project1/ConstantBSProcess.cpp

```diff
@@ -0,0 +1,40 @@
+#include <ConstantBSProcess.h>
+#include <ql/stochasticprocess.hpp>
+#include <ql/termstructures/volatility/equityfx/localvolsurface.hpp>
+#include <ql/termstructures/volatility/equityfx/localvolcurve.hpp>
+#include <ql/termstructures/volatility/equityfx/localconstantvol.hpp>
+#include <ql/termstructures/yieldtermstructure.hpp>
+
+using namespace std;
+
+namespace QuantLib {
+
+ConstantBSProcess::ConstantBSProcess(const Handle<Quote> S,const Handle<YieldTermStructure>& r,const Handle<BlackVolTermStructure>& v,const Handle<YieldTermStructure>& q, Date maturity,Real strike,
+                boost::shared_ptr<discretization>& d )
+                : StochasticProcess1D(d){
```

Comment:
You're not saving S, that you need in the `x0()` method.
```suggestion
                : StochasticProcess1D(d), S(S) {
```